### PR TITLE
EEC-5: Summary page

### PR DESCRIPTION
**Pull request by jamiecarterHO**
## What?

Add the summary page for the eVisa error correction form.
Added as it is shown in the UCD

## Why?

[EEC-5](https://collaboration.homeoffice.gov.uk/jira/browse/EEC-5)

## How?

Added `summary-data-sections.js` and necessary parameters to `pages.json`. Added views/partials
Updated other files to work with summary requirements. 

## Testing?

Updated tests for `utils.getLabel()`

## Check list

- [x] I have reviewed my own pull request
- [x] I have written tests (if relevant)

### CODE REVIEW

File: apps/eec/views/partials/summary-table.html
[the remark this answers is not in the document]
Comment (jamiecarterHO) [reply apps/eec/views/partials/summary-table.html:1]: Removed template, added additional class to make section label text bold.

File: apps/eec/translations/src/en/pages.json [hunk 1]
```diff
@@ -15,12 +15,60 @@
...
   },
   "request-sent": {
-    "header": "Request sent"
+    "header": "Request sent",
+    "send-request": {
+      "header": "Send your request",
+      "disclaimer": "By sending your request you are confirming that, to the best of your knowledge, the details you are providing are correct.",
+      "link-intro": "Find out",
+      "link-text": "how the Home Office uses your personal information"
+    }
```
[the remark this answers is not in the document]
Comment (jamiecarterHO) [reply apps/eec/translations/src/en/pages.json:21]: moved to correct section and accompanying references updated

File: apps/eec/sections/summary-data-sections.js [hunk 2]
```diff
@@ -78,6 +78,24 @@ module.exports = {
         }
       }
     ]
-
+  },
```
Comment (dk4g) [inline apps/eec/sections/summary-data-sections.js:81]: `/someone-else` step with`completing-for-someone-else` field is missing

**Review by dk4g — CHANGES_REQUESTED** [review]

File: apps/eec/sections/summary-data-sections.js (hunk 2 above)
Comment (jamiecarterHO) [reply apps/eec/sections/summary-data-sections.js:85]: Added

**Review by dk4g — APPROVED** [review]

**Pull request merged**